src/notifications/didChangeConfiguration.ts

```diff
@@ -1,11 +1,29 @@
-export function sendDidChangeConfigurationNotification(client, novaConfig) {
+interface ElixirLS {
+  dialyzerEnabled: boolean;
+  dialyzerFormat: string;
+  mixEnv: string;
+  projectDir?: string;
+}
+
+export function sendDidChangeConfigurationNotification(client: LanguageClient, novaConfig: Configuration) {
+  let elixirLS: ElixirLS = {
+    dialyzerEnabled: Boolean(novaConfig.get("elixir-ls.dialyzerEnabled")),
+    dialyzerFormat: "dialyzer",
+    mixEnv: String(novaConfig.get("elixir-ls.mixEnv")),
+  }
+  
+  const projectDir = novaConfig.get("elixir-ls.projectDir");
+  
+  if (projectDir) {
```

Comment:
With the default set we should be able to remove this conditional and include `projectDir` with the other options.